hassio/secrets.py

```diff
@@ -55,6 +55,10 @@ def get(self, secret: str) -> str:
             yaml = YAML()
             data = await self.sys_run_in_executor(yaml.load, self.path_secrets) or {}
 
+            # Filter to only get supported values
+            # pylint: disable=unidiomatic-typecheck
+            data = {k: v for k, v in data.items() if type(v) in (int, float, str)}
+
```

Comment:
I'm fine to use bool also. Not sure if `isInstance(v, (int, float, str))` solve you lint issue

Reply:
Yes, that will solve it.

Reply:
Done.